syntax = "proto3";
package system_map;





/*
UpdateSystem请求
*/
message UpdateSystemRequest {
    /*
    应用系统ID
    */
    string systemId = 1 ;
    /*
    应用系统 ID
    */
    string instanceId = 2 ;
    /*
    名称
    */
    string name = 3 ;
    /*
    链接地址
    */
    string url = 4 ;
    /*
    图标
    */
    string icon = 5 ;
    /*
    类型
    */
    string category = 6 ;
    /*
    描述
    */
    string memo = 7 ;
}

/*
UpdateSystem返回
*/
message UpdateSystemResponse {
    /*
    应用系统 id
    */
    string instanceId = 1 ;
}

/*
UpdateSystemApi返回
*/
message UpdateSystemResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    system_map.UpdateSystemResponse data = 4 ;
}

